src/lib/getSchemaUi.ts

```
@@ -0,0 +1,228 @@
+      description: `Circular reference to **${circularRef}**`,
+      meta: { isCircularReference: true },
+    }
+  }
+
+  static createOneOfProperty(oneOfProperties: Partial<OpenAPI.SchemaObject>[], name: string = ''): OAProperty {
+    return {
+      name,
+      types: ['object'],
+      required: false,
+      properties: oneOfProperties.map((prop) => {
+        const property = this.schemaToUiProperty('', prop)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)